mLRS_Flasher.py

```diff
@@ -1095,10 +1108,28 @@ def flashTxModuleInternalFirmware(self):
         print('ERROR: flashTxModuleInternalFirmware() [2]')
 
     def flashTxModuleInternalWirelessBridgeFirmware(self):
+        #print('flashTxModuleInternalWirelessBridgeFirmware()')
+        device_type = self.fTxModuleInternal_DeviceType_menu.get()
+        firmware_filename = self.fTxModuleInternal_FirmwareFile_menu.get()
+        if 'failed' in firmware_filename:
+            print('ERROR: flashTxModuleEInternalWirelessBridgeFirmware() [1]')
+            return
+        print(device_type, firmware_filename)
```

Comment:
we rally want it or it's just for debug?
I guess one would want to double check with the other places where this likely came from